### ISSUE TITLE
Paint Tray slight design change

### ISSUE THREAD

**Issue by RoYoMi** [florianfesti/boxes#106]
Would be nice to have a horizontal brace to tie the top and bottom together. This would make the assembly a bit more like an "i beam" and would prevent the shelf from sagging over time. 

I added notches in the dead zone between bottles. Of course, this mostly works will with non-hex nesting. 

I would also like something on the exposed edges just to prevent the top layer from wiggling. 
![2018-12-31_23-55-33](https://user-images.githubusercontent.com/8366040/50570629-d5b16780-0d57-11e9-9796-1661816436f5.png)

**Comment by florianfesti** [comment]
Yeah, I've actually already suggested such a change while discussing the PR (https://github.com/florianfesti/boxes/pull/96#issuecomment-433697272). The hex pattern makes implementing this a bit tricky. It also need to check if there is enough room in between the bottles.

May be it is enough to connect the two layers at some points. While this won't have the I beam effect it will still create a wire frame beam that should be much stiffer than the bottom layer alone.
